feature/home/presentation/viewmodel/src/main/java/com/chocolatecake/viewmodel/people/PeopleDetailsListener.kt

```diff
@@ -0,0 +1,10 @@
+package com.chocolatecake.viewmodel.people
+
+import com.chocolatecake.bases.BaseInteractionListener
+
+interface PeopleDetailsListener : BaseInteractionListener {
+    fun onClickMovies(itemId: Int)
+    fun onClickTvShows(itemId: Int)
+    fun backNavigate()
+
+}
```

Comment:
make PeopleDetailsListener  implement media listener and then make the viewmodel implement PeopleDetailsListener